cmd/ocm/create/cluster/cmd.go

```diff
@@ -383,6 +392,14 @@ func preRun(cmd *cobra.Command, argv []string) error {
 		return err
 	}
 
+	args.existingVPC.Enabled = false
+	args.clusterWideProxy.Enabled = false
```

Comment:
NP: no need to explicitly set these to `false`, that is the zero value of bool

Reply:
correct. Thanks!